src/app/screens/confirmBrc20Transaction/confirmBrc20TransactionComponent.tsx

```diff
@@ -0,0 +1,236 @@
+import ActionButton from '@components/button';
+import BigNumber from 'bignumber.js';
+import Brc20FeesComponent from '@screens/confirmBrc20Transaction/brc20FeesComponent';
```

Comment:
nit: For consistency, would prefer:
```suggestion
import Brc20FeesComponent from './brc20FeesComponent';
```